src/AmqpBundle/DependencyInjection/Configuration.php

```diff
@@ -27,7 +27,16 @@ public function getConfigTreeBuilder()
                 ->booleanNode('debug')->defaultValue('%kernel.debug%')->end()
                 ->booleanNode('event_dispatcher')->defaultTrue()->end()
                 ->booleanNode('prototype')->defaultFalse()->end()
-                ->booleanNode('sandbox')->defaultValue(false)->end()
+                ->arrayNode('sandbox')
+                    ->prototype('array')
```

Comment:
My bad. Changed. Also found a problem after moving `(bool) sandbox` to `(bool) sandbox.enabled)`. Fixed + added test case reproduce.

Should I rebase after your review?